src/applications/ivc-champva/shared/utilities.js

```diff
@@ -142,14 +142,14 @@ export function adjustYearString(data) {
 /**
  * Combine all street fields from an address into a single string.
  * @param {Object} addr Standard form address object containing one or more `street` properties (e.g., street, street1, street2)
- * @returns String of all street fields combined.
+ * @returns Copy of passed-in address object with a new `streetCombined` property (string)
  */
 export function concatStreets(addr) {
-  let res = '';
+  const updated = { ...addr, streetCombined: '' };
   if (addr) {
     for (const [k, v] of Object.entries(addr)) {
-      res += k.includes('street') ? `${v} ` : '';
+      updated.streetCombined += k.includes('street') ? `${v} ` : '';
     }
   }
-  return res;
+  return updated;
 }
```

Comment:
Update function so that rather than taking a whole address and only returning the `streetCombined` string, we return the entire address object back PLUS the `streetCombined` property.